scripts/pi-hole/php/queryads.php

```diff
@@ -47,15 +49,16 @@ function echoEvent($datatext)
     exit;
 }
 
+$options = '';
 if (isset($_GET['exact'])) {
-    $exact = '-exact';
-} elseif (isset($_GET['bp'])) {
-    $exact = '-bp';
-} else {
-    $exact = '';
+    $options .= ' -exact';
+}
+
+if (isset($_GET['showall'])) {
+    $options .= ' -all';
 }
 
-$proc = popen('sudo pihole -q -adlist '.$url.' '.$exact, 'r');
+$proc = popen('sudo pihole -q -adlist '.$url.$options, 'r');
```

Comment:
```suggestion
$proc = popen('sudo pihole -q '.$url.$options, 'r');
```

This is not needed. `-adlist` does nothing.

Reply:
Strange... It's working here.

Reply:
Ok I tested again with mobile... Working now. Maybe a cache in issue. I'll try tmw at the PC again.

Reply:
After clearing cache it's working now.